duckduckgo/macos-browser
# Phishing Detection Feature Preferences View

Task/Issue URL: https://app.asana.com/0/1199230911884351/1208149630394246/f
Tech Design URL: https://app.asana.com/0/481882893211075/1207220724600204/f
CC:

**Description**:
Implement GeneralPreferencesView for Phishing Detection Error Page. Also add feature flag for enabling/disabling the preferences view.

**Steps to test this PR**:
1. Go to Settings>General
2. Check there is a Malicious Site Protection section
3. Turn it off
4. Visit privacy-test-pages.site/security/badware/phishing.html
5. Check error page is not shown
6. Turn it back on
7. Retest, ensure the error page is shown



**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
